docs/source/_static/switcher.json

```diff
@@ -1,10 +1,6 @@
 [
     {
         "version": "latest",
-        "url": "https://czbiohub/main/index.html"
-    },
-    {
-        "version": "0.1.0dev3",
-        "url": "https://mysite.org/en/v0.1.0dev3/index.html"
+        "url": "https://czbiohub.github.io/iohub/main/index.html"
```

Comment:
again it would be nice to avoid the hard-coded absolute URL here, but I can imagine that that might require dynamically generating this json file in `conf.py` or else injecting the URL into this file in the github action, and I can see that that might be too complicated/opaque

Reply:
I *think* I can get away with an absolute path without the domain name.